docs/source/static/owkin.css

```diff
@@ -1,97 +1,181 @@
-@import url("https://fonts.googleapis.com/css2?family=Didact+Gothic&display=swap");
-@import url("https://fonts.googleapis.com/css2?family=Didact+Gothic&family=Inter:wght@100;200;300;400;500;600;700;800;900&display=swap");
 
-:root {
-    --color-gray-500: #7f7f7f;
-    --color-teal: #32c6c6;
-    --color-blue: #37b3ff;
-    --color-gold: #ffc200;
-    --color-lightest-gray: #f5f5f5;
-    --color-light-gray: #d9d9d9;
-    --color-light-gold: #faf3eb;
+/* VARIABLES */
+
+/* Font face */
+
+@font-face {
```

Comment:
To be iso with what we have  as font-weights in frontend, should be light: 100, regular: 400, medium: 500, bold: 700

Reply:
You are right! I'm on it!

Reply:
Seems to be good now :)